src/scraper.py

```diff
@@ -1,20 +1,67 @@
+# -*- coding: utf-8 -*-
```

Comment:
Using this coding comment is only required for Python 2 I guess.

Reply:
Good to know. It's about time I fully switch to Python3.
For everyone interested one can find it in [PEP263](https://www.python.org/dev/peps/pep-0263/)